TESTS.md

```diff
@@ -0,0 +1,32 @@
+# Tests
+
+## Running Tests
+
+`go test` is used for testing.  A running PostgreSQL server is required, with
+the ability to log in.  The database to
```

Comment:
can you do something to fix this line wrapping? either fmt or one sentence per line. i don't care.

Reply:
I'm happy to fix it however is needed, but it's not clear how it should be formatted based off the existing code. Right now I formatted it as wrap at 80 chars and I retained the two character spacing between sentences. How should I fix it?

Reply:
```
`go test` is used for testing. A running PostgreSQL
server is required, with the ability to log in. The
database to connect to test with is "pqgotest," on
"localhost" but these can be overridden using [environment
variables](https://www.postgresql.org/docs/9.3/static/libpq-envars.html).
```